File: asion-cloud/asion-spring-boot-admin/src/main/java/org/asion/cloud/server/AsionSpringBootAdminApplication.java
package org.asion.cloud.server;

import de.codecentric.boot.admin.config.EnableAdminServer;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.eureka.EnableEurekaClient;

@SpringBootApplication
@EnableEurekaClient
@EnableAdminServer
public class AsionSpringBootAdminApplication {

    public static void main(String[] args) {
        SpringApplication.run(AsionSpringBootAdminApplication.class, args);
    }
}

File: asion-cloud/asion-spring-boot-admin/src/main/java/org/asion/cloud/server/AsionSpringBootAdminApplication.kt
package org.asion.cloud.server

import de.codecentric.boot.admin.config.EnableAdminServer
import org.springframework.boot.SpringApplication.run
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cloud.netflix.eureka.EnableEurekaClient

@SpringBootApplication
@EnableEurekaClient
@EnableAdminServer
open class AsionSpringBootAdminApplication

fun main(args: Array<String>) {
    run(AsionSpringBootAdminApplication::class.java, *args)
}
